wordpress-mobile/WordPress-iOS
# Blaze Manage Campaigns: Use ui_status to display campaign status

Related Issue #20745

- Update both the Dashboard card and the Campaign List to use the new `ui_status` field that moves the business logic for displaying the campaign statuses to backend

## To test:

There is no good way to test it currently until we integrate the backend (coming soon). It's a bit too early.

To see the new campaign card, follow the instructions from #20932.
To change the mocked data, update the `mockResponse` in `DashboardBlazeCampaignsCardView.swift`.

## Regression Notes
1. Potential unintended areas of impact: only the new Blaze Campaign card and list
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual tests
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:
